client/src/reducers/store.tsx

```diff
@@ -0,0 +1,19 @@
+import { combineReducers, Action, AnyAction } from "redux";
```

Comment:
fix file structure here as well
all redux should be under src/redux
and then you have reducers folder, actions folder and a store file.